app/controllers/content_controller.rb

```diff
@@ -16,7 +16,7 @@ def show
     @markdown = GovspeakToHTML.new.translate_markdown(@page.markdown)
 
     # If in a section page and page has parent, it must be a child of existing page
-    not_found if content_section? && params["section"] != parent_slug
+    not_found if (@page.parent || content_section?) && params["section"] != parent_slug
```

Comment:
Ideally we should to add tests for routing changes

Reply:
Agreed.  Need to think about this a bit.